mathjax3-ts/input/tex/unicode/UnicodeConfiguration.ts

```diff
@@ -0,0 +1,101 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the unicode package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import {EnvList} from '../StackItem.js';
+import TexParser from '../TexParser.js';
+import TexError from '../TexError.js';
+import {TexConstant} from '../TexConstants.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import ParseUtil from '../ParseUtil.js';
+import NodeUtil from '../NodeUtil.js';
+import {numeric} from '../../../util/Entities.js';
+
+// Namespace
+export let UnicodeMethods: Record<string, ParseMethod> = {};
+
+let UnicodeCache: {[key: number]: [number, number, string, number]} = {};
```

Comment:
This should probably by a TeX-input-jax-specific object rather than a global object.  (Since it is set based on user input, if you had two TeX input jax for different documents, the information would bleed from one to the other through this global.)  Unfortunately, this is not currently very easy, as the configuration objects don't have a means of specifying parser-specific data.  Other packages (like color and require) use the parser `options` object to store these, but it is a hack.

It's probably OK to leave this for now, but we need a more general solution to the parser-specific data problem for the future.

Reply:
Alternative, we could make the cache a `Map<[number, string], [number, number, string, number]>` wrt. `[key, font]` pairs. Then bleed through should not matter.
That would, however mean that we would have use define the pairs as objects and give them a `Symbol.iterator`, which might be overkill.
Anyway, for now added to issue #166.